### PR TITLE
[bitnami/rabbitmq-cluster-operator] feat: add revisionHistoryLimit

### PR DESCRIPTION
**Pull request by KyriosGN0**
### Description of the change

Added revisionHistoryLimit parameter for cluster-operator and message topology operator

### Benefits

Users will be able to set this parameters to control the number of replicasets to keep in k8s

### Possible drawbacks

None
### Applicable issues
None
### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by KyriosGN0** [comment]
hey @andresbono , could you take a look at my PR ? 
is there something missing?

**Comment by KyriosGN0** [comment]
hey @andresbono, any updates on mr PR ? 
thanks in advance !

File: bitnami/rabbitmq-cluster-operator/values.yaml [hunk 1]
```diff
@@ -131,6 +131,9 @@ clusterOperator:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+  ## @param clusterOperator.revisionHistoryLimit sets number of replicaset to keep in k8s
+  ##
+  revisionHistoryLimit: 3
```
Comment (andresbono) [inline bitnami/rabbitmq-cluster-operator/values.yaml:136]: What if we honor the Kubernetes default value which is `10`?
```suggestion
  revisionHistoryLimit: 10
```

File: bitnami/rabbitmq-cluster-operator/values.yaml [hunk 2]
```diff
@@ -661,6 +664,9 @@ msgTopologyOperator:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+  ## @param msgTopologyOperator.revisionHistoryLimit sets number of replicaset to keep in k8s
+  ##
+  revisionHistoryLimit: 3
```
Comment (andresbono) [inline bitnami/rabbitmq-cluster-operator/values.yaml:669]: What if we honor the Kubernetes default value which is `10`?
```suggestion
  revisionHistoryLimit: 10
```

File: bitnami/rabbitmq-cluster-operator/Chart.yaml [hunk 3]
```diff
@@ -37,4 +37,4 @@ maintainers:
 name: rabbitmq-cluster-operator
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq-cluster-operator
-version: 4.0.0
+version: 4.0.1
```
Comment (andresbono) [inline bitnami/rabbitmq-cluster-operator/Chart.yaml:40]: Let's bump the minor as this PR is introducing new parameters

**Review by andresbono — COMMENTED** [review]
Thank you for your contribution!! Could you please check my suggestions?

**Comment by KyriosGN0** [comment]
@andresbono nuked my git, will attempt to fix and ping you

**Comment by KyriosGN0** [comment]
nvm will reopen the PR again as my fork is messed up

**Pull request closed without merging**